src/Facebook/InstantArticles/Elements/RelatedArticles.php

```diff
@@ -126,4 +130,35 @@ public function toDOMElement($document = null)
 
         return $element;
     }
+
+    /**
+     * Overrides the @see Element::isValid().
+     *
+     * @return true for valid RelatedArticles that contains all RelatedItem's valid, false otherwise.
+     */
+    public function isValid()
+    {
+        foreach ($this->items as $item) {
```

Comment:
Same thing as for Lists here. If any related article is valid, let's consider the container valid. Then  only the invalid children are omitted.